website/blog/2024-01-24-release-1.7.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Release Notes - Podman Desktop 1.7
+description: Podman Desktop 1.7 has been released!
+slug: podman-desktop-release-1.7
+authors: deboer
+tags: [podman-desktop, release, kubernetes, openshift, onboarding, extensions, settings]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.7/renovations.png
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.7 Release! 🎉
+
+![Podman-desktop-1-7-hero](img/podman-desktop-release-1.7/renovations.png)
+
+This release introduces:
```

Comment:
"We've got a new release with a ton of seal appeal! This release includes:"

Reply:
Updated in new commit.